config_podinit.go

```diff
@@ -46,13 +47,18 @@ func (c *PodInitConfig) Set() error {
 
 	_, err := coll.UpdateOne(ctx, byId(c.SectionId()), bson.M{
 		"$set": bson.M{
-			podInitS3BaseURLKey: c.S3BaseURL,
+			podInitS3BaseURLKey:       c.S3BaseURL,
+			maxParallelPodRequestsKey: c.MaxParallelPodRequests,
 		},
 	}, options.Update().SetUpsert(true))
 
 	return errors.Wrapf(err, "error updating section %s", c.SectionId())
 }
 
 func (c *PodInitConfig) ValidateAndDefault() error {
+	if c.MaxParallelPodRequests <= 0 {
+		// TODO: Determine empirically if this is indeed reasonable
```

Comment:
Can you add a Jira ticket to act on this TODO?

I would set the default if it's equal to 0, but error if the request limit is negative.

Reply:
Jira ticket is added. We do the same `<=` thing here in hostinit, so I think we should not error on negatives.

Reply:
I would argue that hostinit is doing the wrong thing because negatives are invalid values. Negatives should not be equivalent to the default because negative numbers tend to be associated with "unlimited" or other connotations. Realistically, if a person ever set the value to negative, it should be a different effect from setting it to 0 (which is generally understood to mean "the default").

I would correct both to make it invalid to have negative values.

Reply:
Sounds reasonable to me -- I've seen -1 used as a sentinel value a lot but rarely as a default.